Scripts/parameters/zone.py

```diff
@@ -93,6 +93,17 @@
     "helsinki_other": (1000, 1999),
     "espoo_vant_kau": (2000, 5999),
     "surrounding": (6000, 15999),
+    "surround_train": (
+        (6000, 6999),
+        (10000, 11999),
+        (13000, 14999),
+        (15500, 15999),
+    ),
+    "surround_other": (
+        (7000, 9999),
+        (12000, 12999),
+        (15000, 15499),
+    ),
```

Comment:
Unfortunately the area definition used in Ramboll indicators is much more complicated and requires substantial code changes in `ZoneIntervals`.

Reply:
I don't understand! Can you elaborate here or in chat?